### ISSUE TITLE
Questions about logits warper

### ISSUE THREAD

**Issue by lgduu** [misonuma/cge#1]
I noticed that you have added two new LogitsWarper

class MinPLogitsWarper(LogitsWarper):

class MinKLogitsWarper(LogitsWarper):

But you haven't used it, what are they for?

**Comment by misonuma** [comment]
These components are not used. I just forgot to remove these components from the final code.
